checks.go

```diff
@@ -102,19 +103,23 @@ func runCheck(cond cond) bool {
 	debug("Running condition:\n", cond)
 
 	not := "Not"
+	regex := "Regex"
 	condFunc := ""
 	negation := false
+	cond.Regex = false
 
 	// Ensure that condition type is a valid length
-	if len(cond.Type) <= len(not) {
+	if len(cond.Type) <= len(regex) {
 		fail(`Condition type "` + cond.Type + `" is not long enough to be valid. Do you have a "type = 'CheckTypeHere'" for all check conditions?`)
 		return false
 	}
 
-	condEnding := cond.Type[len(cond.Type)-len(not) : len(cond.Type)]
-	if condEnding == not {
+	if cond.Type[len(cond.Type)-len(not):len(cond.Type)] == not {
 		negation = true
 		condFunc = cond.Type[:len(cond.Type)-len(not)]
+	} else if cond.Type[len(cond.Type)-len(regex):len(cond.Type)] == regex {
```

Comment:
Does this prevent doing RegexNot commands since it's an else if?

Reply:
Yeah it does currently -- my next push should check for and remove `Not`, then check for `Regex` and remove it